### PR TITLE
Add IAA model

### PR DESCRIPTION
**Pull request by orenyk**
Moves 7600A fields out of the accounts table and into its own model.
Required some refactoring of views and seed script to reflect the new
schema.

This intentionally leaves a bunch of views and controllers broken. I
will be revamping the schema over the next few commits and then
installing administrate as an initial mechanism for managing the data.

Blocked on #3

### CODE REVIEW

**Comment by orenyk** [comment]
Add dashboard for IAAs.

File: app/models/iaa.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+class Iaa < ApplicationRecord
+  belongs_to :account
+
+  validates :number, presence: true,
+                     uniqueness: { case_sensitive: false }
+  validates :estimated_amount, numericality: { greater_than_or_equal_to: 0 }
+  validates :billed_amount, numericality: { greater_than_or_equal_to: 0 }
+  validates :active, inclusion: { in: [true, false] }
```
Comment (orenyk) [inline app/models/iaa.rb:8]: This can be removed

**Review by achapm — APPROVED** [review]
Approved pending commented changes

**Review by achapm — APPROVED** [review]
LGTM

**Pull request merged**